meta-resin-common/recipes-containers/docker/docker/docker.service

```diff
@@ -1,8 +1,8 @@
 [Unit]
 Description=Docker Application Container Engine
 Documentation=http://docs.docker.com
-After=network.target docker.socket
-Requires=docker.socket
+After=network.target docker.socket var-lib-docker.mount
+Requires=docker.socket var-lib-docker.mount
 
```

Comment:
I have added this inside this commit as this encompasses the addition of our specific var-lib-docker mount point. 
The rce.mount dep is not needed anymore docker will take care of that now that it is not named rce. That was use just to rename the expected name of the tmp mount point.